Light Vox Engine/Engine.cpp

```diff
@@ -133,10 +134,8 @@ LV_RESULT Engine::InitSystems()
     }
 
     //DEBUG:: INTIALIZE ENTITY POSSITIONS
-    static int count = static_cast<int>( sqrtf( static_cast<float>( LV_MAX_INSTANCE_COUNT ) ) );
-    static float rotation = 0.001f;
     {
-
+        static int count = static_cast<int>( sqrtf( static_cast<float>( LV_MAX_INSTANCE_COUNT ) ) );
```

Comment:
`sqrt` is gross but this is an init function so it should be fine. All this casting could produce weird results with a non-square number though.